src/sentry/incidents/logic.py

```diff
@@ -1103,29 +1101,37 @@ def update_alert_rule_trigger_action(
         if type in AlertRuleTriggerAction.INTEGRATION_TYPES:
             integration = updated_fields.get("integration", trigger_action.integration)
             organization = trigger_action.alert_rule_trigger.alert_rule.organization
-            channel_id = get_alert_rule_trigger_action_integration_object_id(
-                type, organization, integration.id, target_identifier,
+
+            target_identifier, target_display = get_target_identifier_display_for_integration(
+                type, target_identifier, organization, integration.id,
```

Comment:
@ceorourke How much work would it be to change the definition of `update_alert_rule_trigger_action` so that the input field for `target_identifier` is `target_value` which is more accurate in the PD case?